wiki/Gameplay/Score/es.md

```diff
@@ -0,0 +1,28 @@
+# Puntuación 
+
+*Para otros usos, consulte: [puntuación (desambiguación)](/wiki/Disambiguation/Score).*
```

Comment:
in notes of all sorts (`For ..., see ...`, `See also: ...`, and so on), article names are usually capitalized -- shouldn't that be the case even in Spanish?
```suggestion
*Para otros usos, consulte: [Puntuación (desambiguación)](/wiki/Disambiguation/Score).*
```

Reply:
Yes, it should be capitalized